packages/web/app/tsconfig.json

```diff
@@ -6,13 +6,7 @@
     "allowJs": true,
     "skipLibCheck": true,
 
-    "strict": false,
-    "useUnknownInCatchVariables": true,
-    "noImplicitThis": true,
-    "strictBindCallApply": true,
-    "alwaysStrict": true,
-    "noImplicitAny": true,
-    "strictFunctionTypes": true,
```

Comment:
@n1ru4l does these options no longer needed with `strict: true`? 🧐

Reply:
yes strict includes all of them